codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/MapConstraintViolationGenerator.kt

```diff
@@ -84,8 +88,23 @@ class MapConstraintViolationGenerator(
                     ${if (keyConstraintViolationExists) "##[doc(hidden)] Key(#{KeyConstraintViolationSymbol})," else ""}
                     ${if (valueConstraintViolationExists) "##[doc(hidden)] Value(#{KeySymbol}, #{ValueConstraintViolationSymbol})," else ""}
                 }
+                
+                impl #{Display} for $constraintViolationName {
+                    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+                        return match self {
```

Comment:
```suggestion
+                        match self {
```
Isn't this caught by Clippy?

Reply:
No, `Clippy` doesn't complain about this. But yes, I'm removing the return and the semicolon.